drivers/mtd_flashpage/mtd_flashpage.c

```diff
@@ -42,16 +44,7 @@ static int _read(mtd_dev_t *dev, void *buf, uint32_t addr, uint32_t size)
 
     (void)dev;
 
-    if (addr % FLASHPAGE_WRITE_BLOCK_ALIGNMENT) {
-        return -EINVAL;
-    }
```

Comment:
@benpicco this one still needs to be re-added for the test to pass, otherwise unaligned reads are still possible.

Reply:
I guess word alignment should suffice here.